examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb

```diff
@@ -7,14 +7,109 @@
    "source": [
     "# Sample Notebook\n",
     "The output of this notebook will be executed via PySpark job then will be stored in the specified notebook path in notebook_args in the Composer DAGs. \n",
+    "\n",
     "This notebook let users \n",
+    "* run a sample Spark Session with a sample CSV file\n",
     "* verify if GCS buckets are properly mounted as a file system and\n",
     "* execute Python files that are stored in mounted GCS buckets by !python and %run commands and\n"
```

Comment:
Do we need to keep this part of the implementation? I mean the gcsfuse and running a python file from the notebook. I thought that you were going to create a separate example to demonstrate that, and keep this project running only PySpark

Reply:
@kristin-kim If you can address the review questions, we can try to get this merged.